tests/Doctrine/Tests/ORM/EntityManagerTest.php

```diff
@@ -204,4 +207,15 @@ public function transactionalCallback($em)
         $this->assertSame($this->_em, $em);
         return 'callback';
     }
+
+    /**
+     * @expectedException \InvalidArgumentException
```

Comment:
Please use `$this->expectExceptionMessage()` and `$this->expectException()` instead of the annotation